Issue #10: Removed default "Hello World" response

mouri11 opened the pull request:
Removed the default "Hello World" reply. Also added /libs/Ab.jar to
resolve dependency conflicts for the AIML bot.

Signed-off-by: Rohit Das <rohit.das950@gmail.com>

mouri11 committed:
MainActivity:Removed default "Hello World" reply

Removed the default "Hello World" reply. Also added /libs/Ab.jar to
resolve dependency conflicts for the AIML bot.

fixes: #10

Signed-off-by: Rohit Das <rohit.das950@gmail.com>

RudraNilBasu commented:
Can you create a patch and send the link as a comment in https://github.com/PhoenixRRDS/AIRYL/issues/10

mouri11 committed:
activity_main: EditText to be at bottom of screen

Changed android:layout_width and android:layout_height of Relative-
Layout to "match_parent". This will ensure the height and width is
adjusted relative to those of the screen.

fixes: #12

Signed-off-by: Rohit Das <rohit.das950@gmail.com>

mouri11 committed:
MainActivity: Logging Chat History

Phase 1: Implementing writing messages and responses to file, named
with current date in (dd-MM-yyyy) format.

fixes: #13

Signed-off-by: Rohit Das <rohit.das950@gmail.com>

mouri11 committed:
Merge branch 'master' into rohitdas

mouri11 committed:
Revert "MainActivity:Removed default "Hello World" reply"

This reverts commit 8498c0ad8e5ec6f4a1ca574e08d3639767c2c198.

mouri11 committed:
MainActivity: Default reply "Hello World" removed

In MainActivity.java, mimicOtherMessage() is used to display bot
reply to a user message. As a default message, "Hello World" was
kept, but on further development, it turned out to be a bane as
it keeps cropping up. So the default reply has been removed.

fixes: #10

Signed-off-by: Rohit Das <rohit.das950@gmail.com>
